examples/cluster/cluster.yaml

```diff
@@ -6,7 +6,11 @@ metadata:
 spec:
   clusterNetwork:
     pods:
-      cidrBlocks: ["192.168.0.0/16"]
+      cidrBlocks:
+        - "10.0.0.0/8"
+    services:
+      cidrBlocks:
+        - "192.168.0.0/16"
```

Comment:
Are these changes necessary and related to this PR?

Reply:
This will break the quick start, which is based on calico today.

Reply:
> This will break the quick start, which is based on calico today.

oho, this is my mistake, this setting just for amazon-vpc-cni. if user want to use calico , the CIDR of pods are different between vpc's.